docs/resources/aws_flow_log.md.erb

```diff
@@ -43,38 +54,39 @@ To search for a flow log by the associated vpc id:
     end
 
 ## Properties
+
 ### flow\_log\_id
 
 The `flow_log_id` property tests the name of the flow log.
 
-  describe aws_flow_log(subnet_id: 'subnet-c6a4319c') do
-    its('flow_log_id') { should cmp 'fl-9c718cf5' }
-  end
+describe aws\_flow\_log(subnet\_id: 'subnet-c6a4319c') do
```

Comment:
I think we need to add indention/backticks and not escape the underscores.

Reply:
There's three different issues here:
1.  ` aws\_flow\_log(subnet\_id` renders improperly. This weirdness/bugginess with middleman (and/or slim). This has been the source of much ongoing pain for us.

Rendered:
<img width="114" alt="screen shot 2018-08-02 at 1 07 55 pm" src="https://user-images.githubusercontent.com/4400151/43608092-22e1b084-9655-11e8-984c-208dbee4ac6c.png">
1. This doesn't render as a codeblock, but the next point will take care of it.
1. We need to go in and put code fences (```ruby) around all the code blocks. Because I need to start enforcing markdown linting in this repo, because of docs things we want to do in the future.

However, since it doesn't lo